modules/osdk-prepare-supported-images.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+:type: Ansible
+:type_lc: ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+:type: Helm
+:type_lc: helm
+endif::[]
+
+:_content-type: PROCEDURE
+[id="osdk-prepare-supported-images_{context}"]
+= Preparing your Operator to use supported images
+
+[IMPORTANT]
+====
+If you are using the Red Hat supported, or downstream, release of the Operator SDK, you do not need to update your project to use supported images as shown in the following procedure. The downstream release of the Operator SDK uses universal base images (UBIs) by default.
+====
+
+Before running your {type}-based Operator on {product-title} Container Platform, update your project to use supported images.
```

Comment:
```suggestion
If your {type}-based Operator does not use universal base images (UBIs), before you run it on {product-title}, update your project to use them.
```

I'd also flip the order of the note and the intro paragraph. I suggest creating and conditionalizing a statement to cover the OKD use case as well.